cmd/lncli/cmd_query_mc_probability.go

```diff
@@ -0,0 +1,70 @@
+// +build routerrpc
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	"github.com/btcsuite/btcutil"
+	"github.com/lightningnetwork/lnd/lnrpc/routerrpc"
+	"github.com/lightningnetwork/lnd/lnwire"
+	"github.com/lightningnetwork/lnd/routing/route"
+	"github.com/urfave/cli"
+)
+
+var queryMcProbCommand = cli.Command{
+	Name:      "querymcprob",
+	Category:  "Payments",
+	Usage:     "Estimate a success probability.",
+	ArgsUsage: "from-node to-node amt",
+	Action:    actionDecorator(queryMcProb),
+}
+
+func queryMcProb(ctx *cli.Context) error {
+	args := ctx.Args()
+
+	if len(args) != 3 {
+		return cli.ShowCommandHelp(ctx, "querymcprob")
+	}
+
+	fromNode, err := route.NewVertexFromStr(args.Get(0))
+	if err != nil {
+		return fmt.Errorf("invalid from node key: %v", err)
+	}
+
+	toNode, err := route.NewVertexFromStr(args.Get(1))
+	if err != nil {
+		return fmt.Errorf("invalid to node key: %v", err)
+	}
+
+	amtSat, err := strconv.ParseUint(args.Get(2), 10, 64)
+	if err != nil {
+		return fmt.Errorf("invalid amt: %v", err)
+	}
+
+	amtMsat := lnwire.NewMSatFromSatoshis(
+		btcutil.Amount(amtSat),
+	)
+
+	conn := getClientConn(ctx, false)
+	defer conn.Close()
+
+	client := routerrpc.NewRouterClient(conn)
+
+	req := &routerrpc.QueryMcProbabilityRequest{
+		NodeFrom: fromNode[:],
```

Comment:
nit: RPC uses `Node(to/from)` and lncli uses `(To/From)Node`, possibly unify to one arrangement for both?

Reply:
swapped